packages/preview-button/admin/src/components/HTMLTemplate/index.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const HTMLTemplate = ({ locale = 'nl', title = 'Preview', children }) => (
+  <html lang={locale}>
```

Comment:
1. Voor compleetheid ook even `dir` toevoegen? `lang` en `dir` gaan hand in hand.
2. Is het niet duidelijker om `lang` te gebruiken ipv `locale`?